recipe/meta.yaml

```
@@ -139,9 +140,10 @@ outputs:
         # Test redirected execve() execution of a cross-compiled binary - Redirected to qemu which executes the binary
         - QEMU_EXECVE=${PREFIX}/bin/qemu-execve-{{ target_emulated_platform }} qemu-execve-{{ target_emulated_platform }} ./execve_call "./hello_from" "EXECVE" > execve_call.txt 2>&1 || true
         - grep "EXECVE" execve_call.txt || exit 1
+        {% endif %}
       requires:
-        - gcc_impl_linux-{{ target_emulated_platform }}
-        - sysroot_linux-{{ target_emulated_platform }} {{ emulated_c_stdlib_version }}.*
+        - gcc_impl_linux-{{ target_emulated_platform }}  # [target_emulated_platform != "riscv64"]
+        - sysroot_linux-{{ target_emulated_platform }} {{ emulated_c_stdlib_version }}.*  # [target_emulated_platform != "riscv64"]
```

at least the sysroot should be possible to include if we override `emulated_c_stdlib_version` to 2.34
```suggestion
        # waiting for https://github.com/conda-forge/ctng-compilers-feedstock/pull/205
        - gcc_impl_linux-{{ target_emulated_platform }}  # [target_emulated_platform != "riscv64"]
        - sysroot_linux-{{ target_emulated_platform }} {{ emulated_c_stdlib_version }}.*
```